pyincore/analyses/mlenabledcgeslc/mlcgeslc.py

```diff
@@ -132,52 +143,56 @@ def run(self) -> bool:
     def get_spec(self):
         return {
             "name": "Salt-Lake-cge",
-            "description": "CGE model for Salt Lake City.",
+            "description": "The “Machine Learning Enabled Computable General Equilibrium (CGE) - Salt Lake City” analysis merges "
```

Comment:
Do we really need the quotation marks within the description?

Reply:
removed quotation marks for safety